### PR TITLE
add isTrusted property in events

### PR DESCRIPTION
**Pull request by sculove**
## Issue
#63 

## Details
- add isTrusted propety in events
 (hold, chagnge, release, animationStart, animationEnd)
- change parameter name from 'inputType' to 'input' for readablility
- change testHelper name from 'TestHeler' to 'TestHelper'
- update sinon version

### CODE REVIEW

File: src/EventManager.ts [hunk 1]
```diff
@@ -27,13 +28,15 @@ export class EventManager {
...
 	 *   // event.inputEvent
+	 *   // isTrusted
 	 * });
 	 */
 	triggerHold(pos: Axis, input: IInputType, event) {
 		this.axes.trigger("hold", {
 			pos,
 			input,
 			inputEvent: event,
+			isTrusted: !!event,
```
Comment (jongmoon) [inline src/EventManager.ts]: Is it possible that `isTrusted` is `false`? If not, If not, is it better to use constant(`true`) to reduce considering means.

**Comment by sculove** [comment]
@jongmoon @happyhj I applied review.

File: src/EventManager.ts [hunk 2]
```diff
@@ -153,37 +156,38 @@ export class EventManager {
...
+		const moveTo = this.am.axm.moveTo(pos);
+		const inputEvent = option && option.event || eventInfo && eventInfo.event || null;
 		const param = {
 			pos: moveTo.pos,
 			delta: moveTo.delta,
-			holding,
-			inputEvent: event,
-			isTrusted: !!event,
-			input,
+			holding: holding,
```
Comment (jongmoon) [inline src/EventManager.ts]: Is it different with previous?
previous
```js
//...
delta: moveTo.delta,
holding, //<-- this one
// ...
```

Comment (sculove) [reply src/EventManager.ts]: the holding property should control in InputObserver.

Comment (sculove) [reply src/EventManager.ts]: you're right! I fit it!

File: declaration/AnimationManager.d.ts [hunk 3]
```diff
@@ -18,25 +18,31 @@ export interface AnimationParam {
...
+        itm: any;
+        em: any;
+        axm: any;
+    });
     getDuration(depaPos: Axis, destPos: Axis, wishDuration?: number): number;
-    private createAnimationParam(pos, duration, inputEvent?);
-    grab(axes: string[], inputType?: IInputType, event?: any): void;
-    restore(inputEvent?: any): void;
+    private createAnimationParam(pos, duration, option);
+    grab(axes: string[], option?: ChangeEventOption): void;
```
Comment (jongmoon) [inline declaration/AnimationManager.d.ts]: `option` is really optional? There's no grab function call in which `option` is omitted

File: src/AnimationManager.ts [hunk 4]
```diff
@@ -73,20 +76,20 @@ export class AnimationManager {
...
-			isTrusted: !!event,
+			inputEvent,
+			input: option && option.input || null,
+			isTrusted: !!inputEvent,
 			done: this.animationEnd
 		};
 	}
 
-	grab(axes: string[], input?: IInputType, event?) {
+	grab(axes: string[], option?: ChangeEventOption) {
```
Comment (jongmoon) [inline src/AnimationManager.ts]: Is `option` is optional?

File: declaration/AnimationManager.d.ts (hunk 3 above)
Comment (sculove) [reply declaration/AnimationManager.d.ts]: Ok you're right. I fit it.

**Comment by jongmoon** [comment]
LGTM

**Pull request merged**